thoth/storages/sync.py

```diff
@@ -41,15 +45,25 @@
 
 
 def sync_adviser_documents(
-    document_ids: list = None, force: bool = False, graceful: bool = False, graph: GraphDatabase = None
+    document_ids: Optional[List[str]] = None,
+    force: bool = False,
+    graceful: bool = False,
+    graph: Optional[GraphDatabase] = None,
+    is_local: bool = False,
 ) -> tuple:
     """Sync adviser documents into graph."""
+    if is_local and not document_ids:
+        raise ValueError(
+            "Cannot sync documents from local directory without explicitly specifying a list of documents to be synced"
+        )
+
     if not graph:
         graph = GraphDatabase()
         graph.connect()
 
-    adviser_store = AdvisersResultsStore()
-    adviser_store.connect()
+    if not is_local:
+        adviser_store = AdvisersResultsStore()
```

Comment:
Is the scope of adviser_store only within this if?!

Reply:
Nope, the variable will exist if the statement is evaluated as true (not always nice, I know...).